app/cypress/e2e/SecvisogramPage/FormEditorTab.cy.js

```diff
@@ -483,4 +483,54 @@ describe('SecvisogramPage / FormEditor Tab', function () {
       ).should('have.value', 'CSAFGID-0001')
     }
   })
+
+  it('should reset product ID counter', function () {
```

Comment:
A similar test should be added for Group Ids...

Reply:
Added a test for group IDs